packages/secrets/package.json

```diff
@@ -39,7 +40,7 @@
     "build": "npm run build:ts && cd src/keyring && napi build --config napi.json --js false --platform --release",
     "build:debug": "cd src/keyring && napi build --config napi.json --js false --platform",
     "build:ts": "tsc --pretty",
-    "install": "node src/keyring/index.js || npm run rebuild",
+    "install": "node scripts/prebuildCheck.js || npm run rebuild",
     "prepack": "node ../../scripts/prepareLicenses.js",
     "prepublishOnly": "bash scripts/prebuildify.sh",
     "rebuild": "npx --yes --package=@napi-rs/cli@2.16.2 -- napi build --config src/keyring/napi.json --cargo-cwd src/keyring --platform --release --js=false src/keyring",
```

Comment:
The "rebuild" script seems to fail with newer versions of npm: `sh: 1: napi: not found`
May be caused by an npm bug so not sure if we can do anything to fix it: https://github.com/npm/cli/issues/6435